chpl/chpl-service/src/main/resources/errors.properties

```diff
@@ -223,3 +269,46 @@ user.notFound=User not found.
 #Warnings
 transparencyAttestation.save=Warning: The transparency attestation for the developer is different in the system than in the upload file. To save the updated transparency attestation, as well as all other developer information, please click 'Save as Developer Information' above before proceeding.
 
+uniqueCHPLId=255
```

Comment:
Needs a comment above to indicate what this section is for, and possibly a prefix to indicate what type of thing we're looking at